src/script/components/MessagesList/Message/ReadIndicator/ReadIndicator.tsx

```diff
@@ -43,10 +43,6 @@ export const ReadIndicator = ({
 }: ReadIndicatorProps) => {
   const {readReceipts} = useKoSubscribableChildren(message, ['readReceipts']);
 
-  if (!message.expectsReadConfirmation) {
-    return null;
-  }
```

Comment:
As far as I can tell, this check was put there *specifically* to *not* show "delivered" if read receipt are disabled on either side.

Removing it does not affect actual read indicator as nothing will be added to the `readReceipt` observable if either of the participants has read receipts turned off